docs/components/carousels/index.html

```diff
@@ -82,7 +82,7 @@
 
             <div class="gel-layout__item gel-2/5@l" style="padding-right: 12px;">
                 <h1 class="gefdocs-header__title">Carousels</h1>
-                <p class="gefdocs-header__summary gel-great-primer">A carousel is a great way to view lots of content in a limited amount of vertical space. It presents a window to view some of a row of content, while indicating there is more out of view that can be scrolled to.</p>
+                <p class="gefdocs-header__summary gel-great-primer">A Carousel is a way to browse lots of content in a limited amount of vertical space, by scrolling a window onto that content</p>
```

Comment:
Is it always vertical space that is limited?

Reply:
@EmmaJP. Hi, Emma. This has been merged, so maybe open an issue.